src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -213,5 +211,21 @@ case class LocalBackend(actorSystem: ActorSystem) extends Backend with SharedFil
     }
   }
 
+  override def callRootPathWithBaseRoot(jobDescriptor: BackendCallJobDescriptor, baseRoot: String): Path = {
+    val path = super.callRootPathWithBaseRoot(jobDescriptor, baseRoot)
+    if (!path.toFile.exists()) path.toFile.mkdirs()
+    path
+  }
+
   override def executionInfoKeys: List[String] = List(InfoKeys.Pid)
+
+  override def callEngineFunctions(descriptor: BackendCallJobDescriptor): CallEngineFunctions = {
+    val workflowDescriptor = descriptor.workflowDescriptor
+    val callRoot = callRootPath(descriptor)
+    val stdout = callRoot.resolve("stdout")
+    val stderr = callRoot.resolve("stderr")
```

Comment:
and if I do a little work on `SgeCallEngineFunctions`, perhaps the line below too